wradlib/georef/raster.py

```diff
@@ -27,37 +27,40 @@
 from osgeo import gdal, osr, gdal_array
 
 from .projection import reproject
+from ..io import dem
 
 
-def pixel_coordinates(nx, ny, mode="centers"):
-    """Get pixel coordinates from a regular grid with dimension nx by ny.
+def pixel_coordinates(nx, ny, edge=False):
```

Comment:
We can't just switch to the new `edge` kwarg since this would introduce a major version bump according to `semver`. We would need to have both kwargs side by side and issue a deprecation warning when using the old one.

Reply:
I could keep the current API.
But what is your general opinion on API change and version in wradlib?

Reply:
- enhancements and non-breaking changes -> increase minor version 
- breaking changes -> increase major version

If we think an API change makes really sense, we can issue deprecation warnings for the old API while accepting old and new API. After some minor version bumps, we can deprecate the old API and release a new major version.